source/_components/deconz.markdown

```diff
@@ -0,0 +1,112 @@
+---
+layout: page
+title: "deCONZ"
+description: "Instructions on how to setup Conbee/Raspbee devices with deCONZ from Dresden Elektronik within Home Assistant."
+date: 2017-11-12 16:30
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: zigbee.png
+ha_category: Hub
+ha_release: "0.58"
+ha_iot_class: "Local Push"
+---
+
+[deCONZ](https://www.dresden-elektronik.de/funktechnik/products/software/pc/deconz/) by [Dresden Elektronik](https://www.dresden-elektronik.de) is a software that communicates with Conbee/Raspbee Zigbee gateways and exposes Zigbee devices that are connected to the gateway.
+
+[deCONZ REST API](http://dresden-elektronik.github.io/deconz-rest-doc/) verified from v2.04.86.
+
+The minimum amount of configuration for deCONZ is the IP ADDRESS, if you don't have the API key. The component will automatically generate an API key for deCONZ. You can later find the API key in deconz.conf inside the home-assistant folder. If you've changed the username and password you can specify those when you configure deconz in configuration.yaml.
+
+You must configure deCONZ by adding the following lines to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+deconz:
+  host: IP ADDRESS
+```
+
+#### {% linkable_title Supported Device types %}
+
+- Zigbee lights
+- Humidity sensors
+- Light Level sensors
+- OpenClose detectors
+- Presence detectors
+- Pressure sensors
+- Switches (remote controls)
+- Temperature sensors
+
+## {% linkable_title Configuration variables %}
+
+- **host** (*Required*): The IP address to your deCONZ server.
+- **api-key** (*Optional*): The API key to your deCONZ server.
+- **port** (*Optional*): Configure port deCONZ web server is accessible from. Default is 80.
```

Comment:
I'm not sure if it's used here but in the more [recent versions](https://github.com/dresden-elektronik/deconz-rest-plugin/issues/238) of deconz you can change the websocket port. Does this need to be taken into consideration here?

Reply:
Websocket port is taken directly from Deconz, no need to configure.